### DISCUSSION TITLE
Netlify Invite Emails go to bare domain not /admin

### DISCUSSION THREAD

**Discussion by arkhamcoffee** [decaporg/decap-cms#7339] [Q&A]
When I invite a user to my site, (Jekyll, source on Github, hosted on Netlify), the email confirmation:

> You have been invited to create a user on (website). Follow this link to accept the invite:
> 
> **Accept the invite**

However the 'Accept the invite' link goes to `/#invite_token=xxx` not `/admin/#invite_token=xxx`

Because the # isn't passed to the domain, I can't use a redirect to fix this. I can't find any way to configure this in Netlify, so is there something in the Decap settings which fixes this?

**Comment by arkhamcoffee** [comment]
Found https://decapcms.org/docs/choosing-a-backend/ which details it's a Netlify thing. Unfortunately Starter (free) accounts can't change the Identity-generated emails.
I presume that's the end of that, unless I want to pay $99 a month.

**Comment by tjgilpin** [comment]
Yep, I've run into the same issue. There's a couple of (pretty clunky TBH!) ways to get around this 

1. Click on 'Accept Invite' - then modify URL in address bar from `/#invite_token=xxx` to `/admin/#invite_token=xxx`
2. Set up a redirect in JS - eg:

```
  const currentHash = window.location.hash;
  if (currentHash.includes("_token")) {
    window.location.href = "/admin/" + currentHash;
  }
```

**Comment by martinjagodic** [comment]
You can define custom Netlify Identity email templates, as documented here: https://docs.netlify.com/security/secure-access-to-sites/identity/identity-generated-emails/#email-templates

Put these files in your repo's admin folder, for instance, `static/admin/emails`. Then you just set the path to each file in Netlify's dashboard.

This is how our templates look like:

**invitation.html**
```html
<h2>You have been invited</h2>

<p>You have been invited to create a user on {{ .SiteURL }}. Follow this link to accept the invite:</p>
<p><a href="{{ .SiteURL }}/admin/#invite_token={{ .Token }}">Accept the invite</a></p>
```

**confirmation.html**
```html
<h2>Confirm your signup</h2>

<p>Follow this link to confirm your user:</p>
<p><a href="{{ .SiteURL }}/admin/#confirmation_token={{ .Token }}">Confirm your mail</a></p>
```

**password-recovery.html**
```html
<h2>Reset Password</h2>

<p>Follow this link to reset the password for your user:</p>
<p><a href="{{ .SiteURL }}/admin/#recovery_token={{ .Token }}">Reset Password</a></p>
```

**email-change.html**
```html
<h2>Confirm Change of Email</h2>

<p>Follow this link to confirm the update of your email from {{ .Email }} to {{ .NewEmail }}:</p>
<p><a href="{{ .SiteURL }}/admin/#email_change_token={{ .Token }}">Change Email</a></p>
```

**Reply by tjgilpin** [reply]
> Put these files in your repo's admin folder, for instance, static/admin/emails. Then you just **set the path to each file in Netlify's dashboard**.

@martinjagodic are you on the pro or enterprise plan? 

On the Free & Starter plan you can change the subject title but not set the path for email templates at https://app.netlify.com/sites/***/configuration/identity#emails
![image](https://github.com/user-attachments/assets/1d87cd21-d97f-4196-b428-afd3a81a968a)

**Reply by arkhamcoffee** [reply]
This is the problem I had. You can't change the templates on a Free & Starter plan.
Decap heavily encourages using Netlify, but the setting users up requires you to have a paid account. Not necessarily unreasonable, but something of a gotcha to find out late in a project.

**Reply by martinjagodic** [reply]
@spade-archer Yes, because Netlify started the project as Netlify CMS, Decap is still best documented to use with Netlify. Now that Decap is an independent open-source project, we want to move to a more agnostic approach.

**Reply by wahidshafique** [reply]
@martinjagodic so what does this mean? Where is the updated documentation. Asking in earnest as someone running into the same issue

**Reply by martinjagodic** [reply]
> Where is the updated documentation

Dwelling in the backlog... Contributions are welcome :)
